doc/manual/src/language/advanced-attributes.md

```diff
@@ -345,3 +347,15 @@ Derivations can declare some infrequently used optional attributes.
     This is useful, for example, when generating self-contained filesystem images with
     their own embedded Nix store: hashes found inside such an image refer
     to the embedded store and not to the host's Nix store.
+
+- [`requiredSystemFeatures`]{#adv-attr-requiredSystemFeatures}\
+
+  If a derivation has the `requiredSystemFeatures` attribute, then Nix will only build it on a machine that has the corresponding features set in its [`system-features` configuration](@docroot@/command-ref/conf-file.md#conf-system-features).
```

Comment:
I think one enables a feature rather than setting it. 

```suggestion
  If a derivation has the `requiredSystemFeatures` attribute, then Nix will only build it on a machine that has the corresponding features enabled in its [`system-features` configuration](@docroot@/command-ref/conf-file.md#conf-system-features).
```

Reply:
"System features" are not something that can be turned on, they are just strings to match between a derivation and a build machine. Their meaning is defined by the user. The special cases listed here are exceptions.